### DISCUSSION TITLE
Tutorial suggestions

### DISCUSSION THREAD

**Discussion by timbwana** [tsunamayo/Starship-EVO#4008] [Suggestions]
### Short Description of the issue:

Appreciate it's early days yet, but the tutorial could benefit from a little love. I had a few issues;

- Right clicking the floor makes it disappear by accident I did this maybe 5 times and fell out of the tutorial habitat.

- I tried to build the floor back, but could not understand how to create a horizontal plane, I could only create vertical planes, this would be a good first tutorial step.

- The first tutorial was to place objects, but did not explain how to rotate them, so I could not place them as suggested, which is when I deleted the floor by accident.

- Building a weapon was a complete mystery, had to link something to something?

- There were a lot of things around in general making it a little obscure which things were needed, maybe separate rooms to compartmentalise the separate tutorial objects?

- The Link Tool was supposed to be behind me at one point to pick up, I had no idea where, somehow managed to discover it on my toolbar by luck.

- The tutorial screens text font sizes are a little too large to fit into their windows so have some missing sentences, had to click to change font sizes. 

### Steps to reproduce the issue:

...

### Environment
- Starship EVO version: 
- Graphics Card: 

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.

**Comment by Uncle-Ulty** [comment]
I had the same problems in the beginning. What helped me were the youtube videos made by players.

**Comment by tsunamayo** [comment]
Thanks a lot for your feedback, all point are very valid and will be addressed - the whole tutorial section will be completely overhauled. 
Cheers
